resources/analysisTools/indelCallingWorkflow/TiN_CanopyBasedClustering.R

```diff
@@ -82,6 +82,16 @@ clusterCentroid <- function (seqType, maxControl=0.45, minTumor=0.01){
 centroid <- clusterCentroid(opt$SeqType, maxControl = as.numeric(opt$rightBorder),
                             minTumor = as.numeric(opt$bottomBorder))
 
+## Testing for enough variants for clustering
+n_germline <- dat %>% filter(Rareness == "Rare") %>% nrow()
+
+cat("No of germline variants: ")
+cat(n_germline, "\n")
+
+if(n_germline <= 50) {
```

Comment:
Do you mean '<' instead of '<=' ? According to below string this seems to be the case...

Reply:
Comment and the code updated to match each other.